utils/get-release-text/package.json

```diff
@@ -8,10 +8,10 @@
     "lint": "eslint src/**/*.ts"
   },
   "dependencies": {
-    "mdast-util-to-string": "^1.1.0",
-    "remark-parse": "^7.0.2",
-    "remark-stringify": "^7.0.4",
-    "unified": "^8.4.2"
+    "mdast-util-to-string": "^2.0.0",
+    "remark-parse": "^9.0.0",
+    "remark-stringify": "^9.0.1",
+    "unified": "^9.2.2"
```

Comment:
I am not sure these updates will work. Last time I have tried to update them, the release page generator stopped working.

Reply:
I just had another look and verified that 
-  `getChangelogEntry()` writes text
- `RELEASE.md` file is generated at the root of the project and is filled

Please also double-check if possible to be 100% sure.

See the screenshot below:

![Screenshot from 2022-08-06 16-13-46](https://user-images.githubusercontent.com/16797721/183252626-6ce59db7-3f5e-47a5-a1f7-f268774d74d9.png)